### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by huuanh99**
## Related Tickets:
- [#44950](https://edu-redmine.sun-asterisk.vn/issues/44950)

## WHAT this PR does?
make schedule work send email for admin show order sale today

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by huuanh99** [comment]
ready

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -13,9 +15,14 @@ class Kernel extends ConsoleKernel
...
      * @return void
      */
+    protected $commands = [
+        OrderCommand::class
+    ];
+
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        // $schedule->command('order:report')->everyMinute();
```
Comment (ghost) [inline app/Console/Kernel.php]: ko dùng xóa đi chứ

**Comment by huuanh99** [comment]
ready

File: app/Console/Kernel.php [hunk 2]
```diff
@@ -13,9 +15,13 @@ class Kernel extends ConsoleKernel
      * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
      * @return void
      */
+    protected $commands = [
+        OrderCommand::class
```
Comment (ghost) [inline app/Console/Kernel.php]:
```suggestion
        OrderCommand::class,
```

**Comment by huuanh99** [comment]
ready

**Pull request merged**